CONTRIBUTING.md

```diff
@@ -803,11 +803,27 @@ Then, [extract strings for the new language](#extracting-new-strings-for-transla
 
    This means it'll register MyDatasource and MyOtherDatasource in superset.my_models module in the source registry.
 
-### Creating a new visualization type
+### Improve visualizations
+
+Superset is working towards a plugin system where new visualizations can be installed as optional npm packages. To achieve this goal,
+we are not accepting pull requests for new community-contributed visualization types at the moment. However, bugfixes for current visualizations are welcome. To edit the frontend code for visualizations, you will have to check out a copy of [apache-superset/superset-ui-plugins](https://github.com/apache-superset/superset-ui-plugins):
+
+```bash
+git clone https://github.com/apache-superset/superset-ui-plugins.git
+yarn && yarn build
+```
+
+Then use `npm link` to create a symlink of the source code in `superset-frontend/node_modules`:
+
+```bash
+cd incubator-superset/superset-frontend
+npm link ../superset-ui-plugins/packages/superset-ui-[PLUGIN NAME]
+
+# Start developing
+npm run dev-server
+```
 
```

Comment:
Thought this might also be worth mentioning...
```suggestion

Note that when plugin packages are linked with `npm link` they will automatically load files from the plugin's `/src` directory.

```